packages/headless/src/controllers/core/did-you-mean/headless-core-did-you-mean.test.ts

```diff
@@ -15,6 +15,17 @@ import {
   DidYouMeanProps,
 } from './headless-core-did-you-mean';
 
+jest.mock('pino', () => ({
+  ...jest.requireActual('pino'),
+  __esModule: true,
+  default: () => ({
+    info: jest.fn(),
+    error: jest.fn(),
+    warn: jest.fn(),
+    debug: jest.fn(),
+  }),
+}));
```

Comment:
future note, maybe we should put that in our root mock or something.
Pino is a PITA to isolate sometimes.